src/ui/public/markdown/markdown.js

```diff
@@ -68,9 +66,13 @@ export class Markdown extends Component {
   }
 
   componentWillReceiveProps(props) {
-    if (props.markdown !== this.props.markdown) {
+    const openLinksInNewTabChanged = props.openLinksInNewTab !== this.props.openLinksInNewTab;
+    if (openLinksInNewTabChanged) {
+      this.markdownIt = markdownFactory(props.whiteListedRules, props.openLinksInNewTab);
```

Comment:
we should probably introduce similar check for whiteListedRules change, as now you can only change them if you also change the openLinksInNewTab prop